release-notes/Cube/Cube_Feature_Release_10.3/Cube_10.3.1.md

```diff
@@ -41,6 +41,16 @@ ChildrenSort="Property|property=Start,asc"
 ChildrenSort="Property|property=End,desc"
 ```
 
+#### Visio - "[GroupValue]" and "[Count]" can now be used as placeholders to adjust the value inside of dynamically grouped shapes [ID_34974]
```

Comment:
- For release notes related to features that need to be configured in Visio, use the "Visual Overview" prefix instead of "Visio", to make it clear that this is a feature in our Visual Overview module instead of Microsoft's Visio application.
- To avoid issues with cross-references, use single quotes in titles.